### PR TITLE
Connect 'core' and 'authentication'.

### PR DESCRIPTION
**Pull request by asynts**
Mirrored at codidact/authentication#10.

Currently, the core and authentication projects do not work together. This pull request changes some configuration such that they work together.

### CODE REVIEW

File: src/WebApp/appsettings.json [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
+            "Microsoft": "Warning",
+            "Microsoft.Hosting.Lifetime": "Information"
+        }
+    },
+    "AllowedHosts": "*",
+    "Identity": {
+        "Authority": "http://localhost:8001",
+        "RequireHttpsMetadata": false,
+        "ClientId": "codidact_client",
+        "ClientSecret": "foo",
```
Comment (luap42) [inline src/WebApp/appsettings.json:17]: Here and in line 3, this PR changed a `~~placeholder~~` value to a concrete value. Is that intentional.

File: src/WebApp/appsettings.json [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
-  "ConnectionStrings": {
-    "DefaultConnection": "--placeholder--"
-  },
-  "Logging": {
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft": "Warning",
-      "Microsoft.Hosting.Lifetime": "Information"
+    "ConnectionStrings": {
+        "DefaultConnection": "Host=localhost;Username=codidact-core;Database=codidact-core"
```
Comment (luap42) [inline src/WebApp/appsettings.json]: Here and in line 17, this PR changed a `~~placeholder~~` value to a concrete value. Is that intentional.

File: src/WebApp/appsettings.json (hunk 1 above)
Comment (asynts) [reply src/WebApp/appsettings.json]: This is a relic of me working on rewriting the documentation. I'll put it back for now.

File: src/WebApp/appsettings.json (hunk 2 above)
Comment (asynts) [reply src/WebApp/appsettings.json]: This is a relic of me working on rewriting the documentation. I'll put it back for now.

Comment (asynts) [reply src/WebApp/appsettings.json]: 17c0831b58236f1d76d6da52eb95928bb7aeec60

File: src/WebApp/appsettings.json (hunk 1 above)
Comment (asynts) [reply src/WebApp/appsettings.json:17]: This was intentional in the authentication project I added the fitting hash of "foo". With a default value it just works out of the box without changes.

[/docs/defaults/src/WebApp/appsettings.json][1]

  [1]: https://github.com/asynts/codidact-authentication/blob/b16dee46e5e1cf5b1db4ad6868d32d3251576c76/docs/defaults/src/WebApp/appsettings.json#L14

**Review by luap42 — APPROVED** [review]

**Pull request merged**